pkg/net/net.go

```diff
@@ -74,6 +75,61 @@ func (o *Overlay) loop(ch <-chan EvtState, state event.Emitter) {
 			}
 		}
 	}()
+
+	for {
```

Comment:
This needs to execute in an separate goroutine, else `New` will block here: 
 https://github.com/wetware/casm/pull/2/files#diff-e68b0087682ee881bcc9e08deeef2431fc56a2b9b37297657cd34f18b1e06d1fR42

Reply:
Okay, will run as a goroutine and add a loop breaker. But [`loop` is called as a goroutine](https://github.com/wetware/casm/blob/83bb01528bc662236b2f624839929bdd1f75a7ac/pkg/net/net.go#L60) right? So, `New` will not block.

Reply:
>But loop is called as a goroutine right?

Yep, my mistake!  🤦‍♂️ 

Disregard the previous comment, it's 👍  as it is.